honeybee_vtk/legend_parameter.py

```diff
@@ -0,0 +1,525 @@
+    original = Colorset.original()
+    peak_load_balance = Colorset.peak_load_balance()
+    shade_benefit = Colorset.shade_benefit()
+    shade_benefit_harm = Colorset.shade_benefit_harm()
+    shade_harm = Colorset.shade_harm()
+    shadow_study = Colorset.shadow_study()
+    therm = Colorset.therm()
+    thermal_comfort = Colorset.thermal_comfort()
+    view_study = Colorset.view_study()
+
+
+class Font:
+    """Fonts for the legend.
+
+        Args:
+            color: A tuple of three integer values for R, G, and B. Defaults to None.
+            size: An integer representing the size of fonts in points. Defaults to None.
+            bold: A boolean to specify whether the fonts should be made bold. If not
+                set, the fonts will not be made bold. Defaults to None.
+        """
+
+    def __init__(
+            self, color: Tuple[float, float, float] = None, size: int = None,
+            bold: bool = None) -> None:
+
+        self.color = color
+        self.size = size
+        self.bold = bold
+
+    @property
+    def color(self) -> Tuple[float, float, float]:
+        """Color of fonts in RGB decimal."""
+        return self._color
+
+    @color.setter
+    def color(self, val) -> None:
+        if not val:
+            self._color = (0, 0, 0)
+        elif isinstance(val, tuple) and _validate_input(val, int, 256):
+            self._color = (val[0]/255, val[1]/255, val[1]/255)
+        else:
+            raise ValueError(
+                'Color accepts a tuple of three integers for R, G, and B values.'
+                f' Instead got {val}.'
+            )
+
+    @property
+    def size(self) -> int:
+        """Size of fonts."""
+        return self._size
+
+    @size.setter
+    def size(self, val) -> None:
+        if not val:
+            self._size = 30
+        elif isinstance(val, int):
+            self._size = val
+        else:
+            raise ValueError(
+                f'Size expects an integer value. Instead got {type(val).__name__}.'
+            )
+
+    @property
+    def bold(self) -> bool:
+        """To make font bold nor not."""
+        return self._bold
+
+    @bold.setter
+    def bold(self, val) -> None:
+        if not val:
+            self._bold = False
+        elif isinstance(val, bool):
+            self._bold = val
+        else:
+            raise ValueError(
+                f'Bold accepts boolean values only. Instead got {val}.'
+            )
+
+    def to_vtk(self) -> vtk.vtkTextProperty:
+        """Create a vtk TextProperty object."""
+        font = vtk.vtkTextProperty()
+        font.SetColor(self._color[0], self._color[1], self._color[2])
+        font.SetFontSize(self._size)
+        if self._bold:
+            font.BoldOn()
+        return font
+
+    def __repr__(self) -> Tuple[str]:
+        return (
+            f'Font color: {self._color} |'
+            f' Font size: {self._size} |'
+            f' Bold: {self._bold}'
+        )
+
+
+class LegendParameter:
```

Comment:
How different is this class from Ladybug's LegendParameter? Can we subclass from that?

Reply:
I knew you'd ask this question. The image below shows correlation. The connected items resemble one another but they are not exact match. Hence, I did not subclass.
![correlation](https://user-images.githubusercontent.com/20086924/121038520-fe15d980-c7cd-11eb-8b96-68d344b83c48.PNG)

Reply:
I feel we might need the rest of the inputs there but I trust you that it's not required for now.